device/transport/mqtt/test/_mqtt_test.js

```diff
@@ -295,6 +295,21 @@ describe('Mqtt', function () {
         fakemqtt.emit('connect', { connack: true });
       });
 
+      /*Tests_SRS_NODE_DEVICE_MQTT_41_004 [ The `sendEvent` method shall call its callback with a `MessageEnqueued` ]*/
+      /*Tests_SRS_NODE_DEVICE_MQTT_41_005 [ The `sendOutputEvent` method shall call its callback with a `MessageEnqueued` ]*/
+      it('calls the callback with the correctly structured response parameter', function (done) {
+        var transport = new Mqtt(fakeAuthenticationProvider, fakeMqttBase);
+        let foo = new results.MessageEnqueued();
+        transport.connect(function () {
+          testConfig.sendFunc(transport, new Message('message'), function(err, resp) {
+            assert.equal(resp.__proto__.constructor.name, "MessageEnqueued");
```

Comment:
>assert.equal(resp.__proto__.constructor.name, "MessageEnqueued"); [](start = 12, length = 65)

Could this be better?   Is there another thing that could be compared here?  Do we  compare objects anywhere else in our unit tests and do it without simply checking the names?